.github/workflows/delete-aws-volumes.yaml

```diff
@@ -0,0 +1,32 @@
+name: Delete unattached (available) EBS volumes in all AWS regions
```

Comment:
I thought it would be better and more useful to have a general script for deleting volumes, in case in the future we want to spin resources in other regions as well. I 'm ok with switching this only to `eu-central-1` if you think it makes more sense though, no strong preference about it.

Reply:
We can add argument to the script with specific region name ... if no argument all regions will be cleared ... what do you think ?

Reply:
Pushing a commit for this!